src/QMCDrivers/VMC/VMCBatched.cpp

```diff
@@ -449,8 +450,8 @@ bool VMCBatched::run()
 
   print_mem("VMCBatched ends", app_log());
 
+  wlog_manager.stopRun();
```

Comment:
Not really an issue, but is there a reason for this change?

Reply:
Driven by https://github.com/QMCPACK/qmcpack/pull/5019/files#r1636930622